CMakeLists.txt

```diff
@@ -654,6 +654,18 @@ if (BUILD_TESTING)
 
     endforeach(test_case)
 
+    if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+        # these tests have 2038 time_t issues
+        set(CTEST_CUSTOM_TESTS_IGNORE
+            s2n_cert_status_extension_test
+            s2n_self_talk_nonblocking_test
+            s2n_session_ticket_test
+            s2n_x509_validator_test
+        )
```

Comment:
Did you create an issue to track removing this?

Reply:
Created the more specific issue of #3948 to track this, and linked it back into the #3903